app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    render json: Message.all
```

Comment:
- [ ] Kindly add to the `Message` table you created 5 different greeting messages as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings.).
### Hint: you can use the `seeds.rb` for adding the greeting messages.

Reply:
Hello @gemmen29 . Thank you for your review. The Table already has five messages I added on my side